src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,61 @@
-export const Tabs = () => {};
+import { Tab } from '../../types/Tab';
+
+type TabsProps = {
+  tabs: Tab[];
+  selectedTabId: string;
```

Comment:
Can we pass prop `isSelected` instead `selectedTabId`? The less the component knows, the better